packages/projects-docs/pages/learn/editors/desktop-setup.mdx

```diff
@@ -69,4 +69,4 @@ You can reset your choice at any time by selecting a different option from the "
 
 ## VS Code Tips and Tricks
 
-To get the most out of VS Code, like automatic extension installation, we recommend you to follow these [tips and tricks](/learn/getting-started/setting-up-repository#vs-code-configuration).
\ No newline at end of file
+To get the most out of VS Code, like automatic extension installation, we recommend you to follow these [tips and tricks](/learn/getting-started/setting-up-repository#vs-code-configuration).
```

Comment:
It looks like this is outdated (the originally referrenced URL seems to be gone) @necoline